### ISSUE TITLE
[Bug] xeyes (And probably other xwayland programs) resizing issue

### ISSUE THREAD

**Issue by AmirDahan** [hyprwm/Hyprland#4159]
### Hyprland Version

a794eecd6a71e431b654cebb1b28dbff0d6da079

### Bug or Regression?

Bug

### Description

When launching xeyes in hyprland and constantly trying to resize it causes hyprland to bug out (Laggy/Freezing)

### How to reproduce

Launch xeyes
Resize it a few times
Drag it around (Whole pc lags)
Keep resizing it and xwayland crashes permanently

### Crash reports, logs, images, videos

output of xeyes after crashing:
X connection to :0 broken (explicit kill or server shutdown).

**Comment by REALERvolker1** [comment]
I just tried this and now all Xwayland apps do not display.

I held right click and dragged on the Xeyes Xwindow and it did start to lag the compositor (I noticed my mouse pointer was very sluggish). Since I launch Xeyes from a .desktop file, I did not see any console output, but when I tried to run it in a terminal, I got no output, and it failed to create the window. I killed the process and then tried out mousepad under forced x11 like `GDK_BACKEND=x11 mousepad`, and likewise, it did not produce a window. Basically what I am trying to say is that this is easy to reproduce.
